modules/psap-using-node-feature-discovery-operator.adoc

```diff
@@ -0,0 +1,175 @@
+
+The Node Feature Discovery (NFD) Operator orchestrates all resources needed to run the Node-Feature-Discovery daemon set by watching for a `NodeFeatureDiscovery` CR. Based on the `NodeFeatureDiscovery` CR, the Operator will create the operand (NFD) components in the desired namespace. You can edit the CR to choose another `namespace`, `image`, `imagePullPolicy`, and `nfd-worker-conf`, among other options.
+
+As a cluster administrator, you can create a `NodeFeatureDiscovery` instance using the {product-title} CLI or the web console.
+
+[id="create-cd-cli_{context}"]
+== Create a NodeFeatureDiscovery instance using the CLI
+
+As a cluster administrator, you can create a `NodeFeatureDiscovery` CR instance using the CLI.
+
+.Prerequisites
+
+* An {product-title} cluster
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the NFD Operator.
+
+.Procedure
+
+. Create the following `NodeFeatureDiscovery` Custom Resource (CR), and then save the YAML in the `NodeFeatureDiscovery.yaml` file:
++
+[source,yaml]
+----
+apiVersion: nfd.openshift.io/v1
+kind: NodeFeatureDiscovery
+metadata:
+  name: nfd-instance
+  namespace: openshift-nfd
+spec:
+  instance: "" # instance is empty by default
+  operand:
+    namespace: openshift-nfd
+    image: quay.io/openshift/origin-node-feature-discovery:4.8
+    imagePullPolicy: Always
+  workerConfig:
+    configData: |
+      #core:
+      #  labelWhiteList:
+      #  noPublish: false
+      #  sleepInterval: 60s
+      #  sources: [all]
+      #  klog:
+      #    addDirHeader: false
+      #    alsologtostderr: false
+      #    logBacktraceAt:
+      #    logtostderr: true
+      #    skipHeaders: false
+      #    stderrthreshold: 2
+      #    v: 0
+      #    vmodule:
+      ##   NOTE: the following options are not dynamically run-time configurable
+      ##         and require a nfd-worker restart to take effect after being changed
+      #    logDir:
+      #    logFile:
+      #    logFileMaxSize: 1800
+      #    skipLogHeaders: false
+      #sources:
+      #  cpu:
+      #    cpuid:
+      ##     NOTE: whitelist has priority over blacklist
+      #      attributeBlacklist:
+      #        - "BMI1"
+      #        - "BMI2"
+      #        - "CLMUL"
+      #        - "CMOV"
+      #        - "CX16"
+      #        - "ERMS"
+      #        - "F16C"
+      #        - "HTT"
+      #        - "LZCNT"
+      #        - "MMX"
+      #        - "MMXEXT"
+      #        - "NX"
+      #        - "POPCNT"
+      #        - "RDRAND"
+      #        - "RDSEED"
+      #        - "RDTSCP"
+      #        - "SGX"
+      #        - "SSE"
+      #        - "SSE2"
+      #        - "SSE3"
+      #        - "SSE4.1"
+      #        - "SSE4.2"
+      #        - "SSSE3"
+      #      attributeWhitelist:
+      #  kernel:
+      #    kconfigFile: "/path/to/kconfig"
+      #    configOpts:
+      #      - "NO_HZ"
+      #      - "X86"
+      #      - "DMI"
+      #  pci:
```

Comment:
@ahardin-rh we need to update the yaml file to uncomment the deviceClassWhitelist and "vendor" in deviceLabelFields.  See https://github.com/openshift/cluster-nfd-operator/blob/master/config/samples/nfd.openshift.io_v1_nodefeaturediscovery.yaml and recent BZ https://bugzilla.redhat.com/show_bug.cgi?id=1966725.

Reply:
I will do that on an upcoming PR, with extra examples requested by @ashishkamra

Reply:
Thanks @wabouhamad @ArangoGutierrez